modules/services/views/src/main/java/au/com/shiftyjelly/pocketcasts/views/component/FileStatusIconsView.kt

```diff
@@ -77,9 +77,12 @@ class FileStatusIconsView @JvmOverloads constructor(
         downloadProgressUpdates: Observable<DownloadProgressUpdate>,
         playbackStateUpdates: Observable<PlaybackState>,
         upNextChangesObservable: Observable<UpNextQueue.State>,
-        hideErrorDetails: Boolean = false
-    ) {
+        hideErrorDetails: Boolean = false,
 
+    ) {
+        val bookmarkVisibilityObservable = BehaviorRelay.create<Boolean?>().apply {
+            accept(episode.hasBookmark)
```

Comment:
Observing changes to the episode's bookmark status will have any effect as the episode is not changing here.